demos/gfql/gfql_cpv_gpu_enchmark.ipynb

```diff
@@ -0,0 +1,2976 @@
+{
```

Comment:
instead of making a new file & url, replace the old file

(and revert any changed links)